src/didmeta.c

```diff
@@ -247,13 +247,13 @@ int DIDMeta_Merge(DIDMeta *meta, DIDMeta *frommeta)
     assert(meta && frommeta);
 
     strcpy(meta->alias, frommeta->alias);
-    if (*frommeta->txid)
+    if (!*meta->txid)
```

Comment:
should we check ("!*meta->txid && !*frommeta->txid) instead of only checking meta->txid.